service/src/main/kotlin/app/cash/backfila/service/persistence/DbRegisteredBackfill.kt

```diff
@@ -66,13 +66,22 @@ class DbRegisteredBackfill() : DbUnsharded<DbRegisteredBackfill>, DbTimestampedE
   @Column
   var requires_approval: Boolean = false
 
+  /**
+   * Notes when this backfill should no longer be useful and the code should be deleted. This will
+   * be used to power removal reminders. It will guarantee that reminders will not occur until
+   * after this date. This defaults to as soon as possible if it is missing.
```

Comment:
hmm is that good behavior? So there's no way to have a backfill that is permanent?

Reply:
Set it 30 years in the future. Is any service truly permanent?

Reply:
:galaxybrain:

Reply:
it is easy enough to push it in the future again. So yeah I think it is fine.